www/_template/getting-started.md

```diff
@@ -3,43 +3,278 @@ layout: layouts/main.njk
+
+Install the canavas-confetti package from npm and use it with the following command:
 
 ```bash
-# Start your dev server, load your site locally
-snowpack dev
+npm install --save canvas-confetti
+```
+
+Now head to `index.html` and add this code to the bottom of the `<body>` tag:
+
+```diff
+   <script type="module" src="/index.js"></script>
++  <script type="module">
++  import confetti from 'canvas-confetti';
++ confetti.create(document.getElementById('canvas'), {
++  resize: true,
++  useWorker: true,
++ })({ particleCount: 200, spread: 200 });
++  </script>
+  </body>
+```
+
+> 💡 Tip: you can also add this code in index.js
+
+You should now see a nifty confetti effect on your site.
+
+IMAGE: Gif showing site loading with a confetti effect
+
+> 💡 Tip: not all NPM modules may work well in the browser. If it's dependent on Node.js built-in modules you'll need to polyfill Node. Read more about how to do this on our [features page.](/features)
+
+## Adding CSS
+
+Snowpack natively supports many file types. CSS and CSS Modules for example. Here you'll add a simple CSS file.
+
+First add the following css as a new `index.css` file:
+
+```css
+body {
+  font-family: sans-serif;
+}
+```
+
+Include it in your project by adding it to index.html in the `<head>`
+
+```diff
+    <meta name="description" content="Starter Snowpack App" />
++   <link rel="stylesheet" type="text/css" href="/index.css" />
+    <title>Starter Snowpack App</title>
+```
+
+Image: GIF showing adding the css to `index.html` and saving, showing the visual changes as the CSS loads.
+
+## Build for production/deployment
+
+OK you've now built the most amazing website ever (or something like that) and you want to launch it. It's time to use `Snowpack build`.
 
-# Build your site for production
-snowpack build
+By default, `snowpack build` builds your site using the same unbundled approach as the `dev` command. Building is tightly integrated with your development setup so that you are guaranteed to get a near-exact copy of the same code that you saw during development.
 
-# Build your site, but watch the file system and rebuild when files change.
-# Great for local development with your own dev server (ex: Rails)
-snowpack build --watch
+Add the `snowpack build` comman to package.json so it's easier to run on the command line:
+
+```diff
+  "scripts": {
+    "start": "snowpack dev",
++   "build": "snowpack build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+
+  },
 
-# See more helpful info
-snowpack --help
 ```
 
-## Install Snowpack from scratch
+Now you can run this in your terminal:
 
 ```bash
-# using npm
-npm install --save-dev snowpack
+npm run build
+```
+
+You should see a new directory called `build` that contains a copy of your Snowpack project ready for deployment.
+
+IMAGE: GIF terminal running Snowpack build, showing output, then clicking on the new `build` directory
+
+## Bundle for deployment
```

Comment:
Thought: what if we move this into its own guide, and then heavily promote it in the "next steps" section.

A belief of the project is that not everyone needs to bundle, even if most want to (especially more complex sites). So that would be more in line with that belief, that bundling isn't a required step to get started but obviously a great next step to read up about, if you care about bundling.